spec/lib/smb2/packet/read_response_spec.rb

```diff
@@ -21,13 +21,20 @@
       expect(packet.struct_size).to eq(17)
     end
 
-    specify 'data' do
+    specify do
       expect(packet.data_length).to eq(92)
+    end
+    specify do
+      expect(packet.data.length).to eq(92)
```

Comment:
If `packet.data_length == packet.data.length` is an invariant that should have an explicit example instead of expecting that both individually are the same value, 92.